crates/types/Cargo.toml

```diff
@@ -33,6 +33,9 @@ sha2          = { workspace = true }
 sha3          = { workspace = true, optional = true }
 thiserror     = { workspace = true }
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+grug-crypto = { path = "../crypto" }
```

Comment:
`MockApi` uses grug-crypto, but it's not wasm-compatible so we need to exclude it from wasm builds.